src/App.tsx

```diff
@@ -1,171 +1,36 @@
-import { Loader } from './components/Loader';
+import './App.scss';
+import {
+  Routes,
+  Route,
+  NavLink,
+  Navigate,
+} from 'react-router-dom';
+
+import { HomePage } from './components/HomePage';
+import { NotFoundPage } from './components/NotFoundPage';
+import { PeoplePage } from './components/PeoplePage';
 
 export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
-        </div>
-      </div>
+  <div className="box">
+    <div className="App">
+      <h1 className="title">People table</h1>
+    </div>
+
+    <nav className="nav">
+      <NavLink to="/">
```

Comment:
If you don't use isActive className, you don't need `NavLink`
```suggestion
      <Link to="/">
```